### DISCUSSION TITLE
Unable to import purchase order date

### DISCUSSION THREAD

**Discussion by ChrisWiltscoll** [snipe/snipe-it#12734] [Q&A]
When we attempt to import in the browser we get error: 

'The purchase date does not match the format Y-m-d'

We have double-checked the format of the date field in CSV using Notepad and it is 100% correct: 

"Item Name,Category,Model Name,Model Number,Manufacturer,Serial Number,Asset Tag,Location,Notes,Purchase Date,Purchase Cost,Supplier,Order number,Invoice number
CC-R000-05,PC,DESKTOP,i7-4930,Stone,,149456,Castle Combe,xxxxxx,2023/11/12,562.4,Stone,67890,IN45678"

We have tried several other date formats all with same result.
We have also removed the date and the import is successful

Any ideas to resolve the issue please?

**Comment by ChrisWiltscoll** [comment]
Corrected CSV Text which we used.......
Item Name,Category,Model Name,Model Number,Manufacturer,Serial Number,Asset Tag,Location,Notes,Purchase Date,Purchase Cost,Supplier,Order number,Invoice number
CC-R000-05,PC,DESKTOP,i7-4930,Stone,,149456,Castle Combe,xxxxxx,2023-11-12,562.4,Stone,67890,IN45678

**Comment by snipe** [comment]
Try upgrading - we fixed a few date problems last week

**Reply by ChrisWiltscoll** [reply]
We are already on the latest version v6.0.14 build 9236 (g05a3f20d5). 
Is there a later version than this?

**Comment by ChrisWiltscoll** [comment]
Hi

Our reply as posted in GitHub:

We are already on the latest version v6.0.14 build 9236 (g05a3f20d5).
Is there a later version than this?

Regards

Chris


[Wiltshire College Logo]

Chris Holt

ICT Systems and Compliance Coordinator



Wiltshire College & University Centre

College Road | Trowbridge | Wiltshire | BA14 0ES

T: 01225 756310 | W: wiltshire.ac.uk<http://www.wiltshire.ac.uk/>

________________________________
